src/extension.ts

```diff
@@ -94,10 +111,27 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 	return Promise.resolve(context);
 }
 
+export async function _stopExistingDfaRun(context: vscode.ExtensionContext, outputChannel: vscode.LogOutputChannel) {
```

Comment:
As a matter of style, typescript functions should have return types declared, so this should be `: Promise<void>`

Reply:
Line 128 has void on it... which is why the compiler is happy. It took me a minute on this one as well... so just for readability/consistency we can remove the void on line 128 and add in : Promise<void> as Josh is suggesting.

Reply:
I will add Promise<void>. I still want to keep line 128 with void in it. Seems like that's the preference for updating caches (by not blocking the original function by using await).